packages/webpack/src/plugin.js

```diff
@@ -464,50 +464,72 @@ class LavaMoatPlugin {
                   )
                   return
                 }
-                diag.rawDebug(
-                  1,
-                  '> adding runtime (additionalChunkRuntimeRequirements)'
+                let runtimeChunks = []
+                if (
+                  chunk.name &&
+                  options.unlockedChunksUnsafe?.test(chunk.name)
+                ) {
+                  diag.rawDebug(
+                    1,
+                    `> adding UNLOCKED runtime for chunk ${chunk.name}`
+                  )
+                  runtimeChunks = [
+                    {
+                      name: 'ENUM',
+                      file: path.join(__dirname, './ENUM.json'),
+                      json: true,
+                    },
+                    {
+                      name: 'runtime',
+                      file: path.join(
+                        __dirname,
+                        './runtime/runtimeUnlocked.js'
+                      ),
+                    },
+                  ]
+                } else {
+                  diag.rawDebug(1, `> adding runtime for chunk ${chunk.name}`)
+                  // narrow down the policy and map to module identifiers
+                  const policyData = identifierLookup.getTranslatedPolicy()
+
+                  runtimeChunks = [
+                    {
+                      name: 'root',
+                      data: identifierLookup.root || null,
```

Comment:
do we mean `||` or `??`?

Reply:
I read this out load "do we mean or or or?"

Reply:
@naugtur which are you concerned about?

Reply:
(opinion) prefer no coercion over coercion.  use `||` if you want coercion; otherwise use `??`